src/components/PDFView/index.native.js

```diff
@@ -169,6 +173,21 @@ class PDFView extends Component {
             </View>
         );
     }
+
+    render() {
+        return this.props.onPress && !this.state.successToLoadPDF ? (
```

Comment:
This condition caused the https://github.com/Expensify/App/issues/42103#issuecomment-2110325553 bug. We should've disabled the component instead of conditional rendering.